packages/react-core/src/components/Alert/AlertActionCloseButton.tsx

```diff
@@ -18,16 +19,19 @@ export const AlertActionCloseButton = ({
   className = '',
   onClose = () => undefined as any,
   'aria-label': ariaLabel = '',
-  title,
```

Comment:
this is also technically a breaking change

Reply:
True, but I don't think anyone was using the `title` prop as an `aria-label` on the `<Button>`.